### ISSUE TITLE
E999 reports invalid column

### ISSUE THREAD

**Issue by asottile** [PyCQA/flake8#169]
In GitLab by @arcanemagus on Oct 17, 2016, 13:24

*Please provide the following version information*

- Version of Python which has Flake8 installed (e.g., `python -V`)

Python 2.7.12 and Python 3.5.1

- How you installed Flake8 (e.g., `pip`, `apt`, `yum`, etc.)

pip

  - If you used pip, the version of pip installed (e.g., `pip --version`)

  v8.1.2

- Version of setuptools installed (e.g., `python -c 'import setuptools; print(setuptools.__version__)'`)

28.6.0

- Version of Flake8 installed (i.e., copy and paste all of the output from `flake8 --version`)

3.0.4 (pycodestyle: 2.0.0, pyflakes: 1.2.3, mccabe: 0.5.2) CPython 2.7.12 on Windows

3.0.4 (pyflakes: 1.2.3, mccabe: 0.5.2, pycodestyle: 2.0.0) CPython 3.5.1 on Windows

----

*Please describe the problem or feature*

Flake8 incorrectly reports the location of syntax errors.

*If this is a bug report, please explain with examples (and example code) what you expected to happen and what actually happened.*

When running on a file with broken syntax, such as this:

```python
foo(
```

Flake8 reports the syntax error as 2 past the last code character. The EOL of the file doesn't seem to matter. The above code (with 4 total characters) is reported as being on character 6 of the line.

```text
.\broken.py:1:1: E902 TokenError: EOF in multi-line statement
.\broken.py:1:6: E999 SyntaxError: unexpected EOF while parsing
```

_Note: Originally reported [here](https://github.com/AtomLinter/linter-flake8/issues/256#issuecomment-254301975)._

**Comment by asottile** [comment]
In GitLab by @lzkelley on Oct 21, 2016, 08:47

This bug is makes `flake8` dependent tools virtually unusable.... *extremely* annoying

**Comment by asottile** [comment]
In GitLab by @sigmavirus24 on Oct 23, 2016, 05:40

Mentioned in merge request !125

**Comment by asottile** [comment]
In GitLab by @sigmavirus24 on Oct 24, 2016, 04:18

Mentioned in commit sigmavirus24/flake8@7fc699e28bda8678d60d7a152a2341781f02220b

**Comment by asottile** [comment]
In GitLab by @sigmavirus24 on Nov 17, 2016, 15:31

Mentioned in merge request !151

**Comment by asottile** [comment]
In GitLab by @sigmavirus24 on Nov 19, 2016, 17:29

Mentioned in commit ae9e832cc1280f3d9d7357baf939bc7e4fa5d9c7

**Comment by asottile** [comment]
In GitLab by @sigmavirus24 on Dec 20, 2016, 17:08

mentioned in commit sigmavirus24/flake8@07c187b8d3e5881ccf85f93a065bf9c2467dcbd1
